settings.gradle.kts

```diff
@@ -52,6 +52,7 @@ include(":graphql-kotlin-spring-server")
 
 // Executions
 include(":graphql-kotlin-transaction-loader-instrumentation")
+include(":graphql-kotlin-dataloader")
```

Comment:
technically order in this file won't matter (as Gradle will figure out the module order based on their dependencies) but since the instrumentation uses dataloader one -> can we move this definition before the instrumentation?